app/src/main/java/com/github/se/icebreakrr/ui/tags/TagView.kt

```diff
@@ -35,6 +36,9 @@ import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.window.PopupProperties
 
+val dropDownMenuTagHeight = 60
+val dropDownMenuMaxNumberTag = 5
+
```

Comment:
This is a nice debug, we needed to make the DropDownMenu smaller, it taked all the screen before, now it is cleaner